### ISSUE TITLE
[BUG] Lighttpd target ignores empty header values

### ISSUE THREAD

**Issue by kenballus** [narfindustries/http-garden#23]
this could use a normalization

**Comment by gstrauss** [comment]
Yes, for simplicity, lighttpd ignores empty request headers.
(Previously discussed in https://redmine.lighttpd.net/issues/3219#note-4)

**Comment by kenballus** [comment]
Yes; I should have made the text of this bug clearer. This is a bug in the Garden that causes unimportant discrepanices to be reported because Lighttpd ignores empty headers, not a bug in Lighttpd that needs to be fixed.
